docs/_coverpage.md

```diff
@@ -1,12 +1,17 @@
+<!-- markdownlint-disable first-line-h1 -->
+
 ![logo](_media/icon.svg)
 
 # docsify <small>4.13.0</small>
 
-> A magical documentation site generator.
+> A magical documentation site generator
 
 - Simple and lightweight
-- No statically built html files
+- No statically built HTML files
```

Comment:
https://github.com/docsifyjs/docsify/tree/v5-style-overhaul?tab=readme-ov-file#features

You can sync changes to README.md

Reply:
Yep.

My plan is do to a separate documentation update before v5 launch. I will make sure the repo's README.md is synced with those changes as part of that PR.